src/Commands/DeployCommand.php

```diff
@@ -32,10 +32,14 @@ public function handle()
     {
         $builderName = $this->option('builder');
         if (is_null($builderName)) {
-            $process = new Process('whoami');
-            $process->run();
-            if ($process->isSuccessful()) {
-                $builderName = trim($process->getOutput());
+            if (class_exists('\Symfony\Component\Process\Process')) {
```

Comment:
You can safely import this class at the top with `use Symfony\Component\Process\Process;`. Import statements in PHP do not have the side effect of autoloading. Then, this would become:

```php
if (class_exists(Process::class)) {
    $process = new Process('whoami');
...
```